src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesDataController.java

```diff
@@ -106,7 +97,57 @@ private ResponseEntity<OverseasEntityDataApi> getOverseasEntityDataResponse(
             overseasEntityDataApi = new OverseasEntityDataApi();
             overseasEntityDataApi.setEmail(email);
         }
-        
+
         return ResponseEntity.ok(overseasEntityDataApi);
     }
+
+    @GetMapping("/managing-officers")
+    public ResponseEntity<ManagingOfficerListDataApi> getManagingOfficers(
+            @PathVariable(TRANSACTION_ID_KEY) String transactionId,
+            @PathVariable(OVERSEAS_ENTITY_ID_KEY) String overseasEntityId,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) throws ServiceException {
+
+        HashMap<String, Object> logMap = createLogMap(transactionId, overseasEntityId);
+        ApiLogger.infoContext(requestId, "Calling service to retrieve the overseas entity submission for overseas entity " + overseasEntityId, logMap);
+
+        final var submissionDtoOptional = overseasEntitiesService.getOverseasEntitySubmission(overseasEntityId);
+
+        if (submissionDtoOptional.isPresent()) {
+            final var submissionDto = submissionDtoOptional.get();
+
+            if (!isRoeUpdateEnabled) {
```

Comment:
Other PR has method for this feature check check. 
Also that PR checks that it's an update submission and says can't when not. Probably better to check but as long as we align the PRs.